README.md

```diff
@@ -36,7 +36,45 @@ Need the copilot to comment, fix a bug, or add a doc string? Simply go to the co
 
 You can log in and out of your Pieces account using the `Pieces: Login` or `Pieces: Logout` command in your command palette. An output panel is available to check your cloud status and login status.
 
-### Connect with the Pieces Community
+### Connect/Disconnect your cloud
+
+You can connect and disconnect from your cloud using  `Pieces: Connect your cloud` or `Pieces: Disconnect your cloud` command in your command palette.(Login required!)
+
+### Search
+
+You can search a saved material using `Pieces: Search`.
+
```

Comment:
TODO go deeper on this in the future, ie w/ different search modes, this will explicitly only search your Snippets/Assets

Reply:
so yes as long as you entering some charaters in the query the results will be shown when you hit enter a list will be available to choose an asset from to open it

Reply:
so yes as long as you entering some characters in the query the results will be shown when you hit enter a list will be available to choose an asset from to open it

Reply:
so yes as long as you entering some charaters in the query the results will be shown when you hit enter a list will be available to choose an asset from to open it